engine/src/main/resources/assets/ui/menu/createGameScreen.ui

```diff
@@ -134,7 +134,19 @@
                         },
                         {
                             "type": "UISpace",
-                            "size": [1, 16]
+                             "size": [1, 16]
+                        },
+                        {
+                            "type": "UILabel",
+                            "text": "Save Game Path:"
```

Comment:
Please integrate this with the translation system, i.e., use `"text": "${engine:menu#save-game-path-label}"` to reference a translation key `save-game-path-label` and add it to the language file(s) in [menu.lang](https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/resources/assets/i18n/menu.lang) and [menu_en.lang](https://github.com/MovingBlocks/Terasology/blob/63e2120cd420ca11c7028d927f96f7214c5d1a71/engine/src/main/resources/assets/i18n/menu_en.lang).